drivers/clk/Kconfig

```diff
@@ -8,11 +8,11 @@ menuconfig CLK
 if CLK
 
 config CLK_RPMSG
-	bool "rpmsg clk driver"
+	bool "RPMGS clk driver"
```

Comment:
It is `RPMSG` instead of `RPMGS`, isn't it?

Reply:
you're right ! fixed